### PR TITLE
More preparations towards creation of a self-contained codeworld server image

### PR DESCRIPTION
**Pull request by nomeata**

### CODE REVIEW

File: gallery-code/codeworld/PEwIFcw4EXZtdP7JdytExKA.hs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+main = collaborationOf(2, initial, step, event, picture)
```
Comment (cdsmith) [inline gallery-code/codeworld/PEwIFcw4EXZtdP7JdytExKA.hs]: I can't accept a contribution to github when a student owns the copyright, without that student agreeing to the CLA and sending the pull request.  I also don't see them contributing much to CodeWorld in the long run.

Can you split these files into a local fork instead?

Comment (nomeata) [inline gallery-code/codeworld/PEwIFcw4EXZtdP7JdytExKA.hs]: Hmm, ok. I will bundle these files with the `Vagrantfile` script that builds the image then instead.

**Comment by nomeata** [comment]
Updated to remove the unwanted files.

**Comment by cdsmith** [comment]
Thanks

**Pull request merged**